plugin/storage/kafka/options.go

```diff
@@ -32,14 +32,28 @@ const (
 	// EncodingZipkinThrift is used for spans encoded as Zipkin Thrift.
 	EncodingZipkinThrift = "zipkin-thrift"
 
-	configPrefix   = "kafka.producer"
-	suffixBrokers  = ".brokers"
-	suffixTopic    = ".topic"
-	suffixEncoding = ".encoding"
-
-	defaultBroker   = "127.0.0.1:9092"
-	defaultTopic    = "jaeger-spans"
-	defaultEncoding = EncodingProto
+	configPrefix         = "kafka.producer"
+	suffixBrokers        = ".brokers"
+	suffixTopic          = ".topic"
+	suffixEncoding       = ".encoding"
+	suffixAuthentication = ".authentication"
```

Comment:
Are there more authentication mechanisms? Is so maybe we should rather have `.kerberos=true`

Reply:
Or `.authentication=kerberos`

Reply:
for now, there is only one authentication mechanism which is in this case Kerberos, but Kafka (and Sarama) support others, I would say that @objectiser suggestion is the way to go, just in case that in the future we wanted more mechanisms.